src/versionedRouter.js

```diff
@@ -481,6 +491,8 @@ async function routerHandleDest(ctx) {
         respEvents.push(...listOutput);
```

Comment:
Can we remove the previous error level references - `TRANSFORMER_METRIC.ERROR_AT.RT`, so that we can see if any additional things to handle might come up ?